specification/recoveryservicessiterecovery/resource-manager/Microsoft.RecoveryServices/stable/2018-07-10/service.json

```diff
@@ -7497,6 +7497,10 @@
           "description": "The recovery availability set Id.",
           "type": "string"
         },
+        "recoveryProximityPlacementGroupId": {
+          "description": "The recovery proximity placement group Id.",
+          "type": "string"
+        },
```

Comment:
Add this property in example as well?

Reply:
Azure Site Recovery (ASR) service supports multiple replication providers and following are published providers (under ReplicationProviderSpecificSettings): * A2AReplicationDetails * HyperVReplicaAzureReplicationDetails * HyperVReplicaBlueReplicationDetails * HyperVReplicaReplicationDetails * InMageAzureV2ReplicationDetails * InMageReplicationDetails
All the resource operations examples are with one such provider "HyperVReplicaAzure".
And the new properties belong to "A2A".
We cannot update the examples.